src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -223,4 +253,26 @@ private static int getStartingLineNumber(String linesChanged) {
 
         return Integer.parseInt(chunkHeaderMatcher.group(STARTING_LINE_NUMBER_GROUP_NAME));
     }
+
+    /**
+     * Returns true if {@code relativePath} exists in the HEAD of the currently checked out commit.
+     * If {@code returnMessage} is empty, it means that the file does not exist in the remote repo.
+     */
+    private static boolean fileExistsInRepo(String relativePath, Path repoRoot) {
+        String returnMessage = GitDiff.diffNumLinesModified(repoRoot, relativePath, Optional.empty(), Optional.empty());
```

Comment:
This method and the following method calls the same git command. In your logic now, this git command will be run twice, which is a waste of resource. Please combine them into one method.